src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyShares.tsx

```diff
@@ -256,9 +263,36 @@ const KeyShareFlow = () => {
       }));
       await Promise.all(promises);
 
+      const validatorsArray: ValidatorType[] = Object.values(validators);
       let currentNonce = ownerNonce;
       let incorrectNonceFlag = false;
-      const validatorsArray: ValidatorType[] = Object.values(validators);
+      let warningTextMessage = '';
+      let maxValidatorsCount = config.GLOBAL_VARIABLE.MAX_VALIDATORS_COUNT_PER_BULK_TRANSACTION;
+
+      const operatorsData: SelectedOperatorData[] = Object.values(operatorStore.selectedOperators).map((operator: IOperator) => {
+        const availableValidatorsAmount = operatorStore.operatorValidatorsLimit - operator.validators_count;
+        let hasError = false;
+
+        if (availableValidatorsAmount < validatorStore.validatorsCount && availableValidatorsAmount !== 0) {
+          warningTextMessage = `The number of validators you wish to onboard would exceed the maximum validator capacity for one of your selected operators. You may proceed with onboarding only ${availableValidatorsAmount} validators.`;
```

Comment:
Please extract this to translations. You can probably set the value there to be:
(availableValidatorsAmount: string) => `The number of validators you wish to onboard would exceed the maximum validator capacity for one of your selected operators. You may proceed with onboarding only ${availableValidatorsAmount} validators.`;

Reply:
+